doc/input-output-reference/src/overview/group-simulation-parameters.tex

```diff
@@ -963,7 +963,7 @@ \subsection{PerformancePrecisionTradeoffs}\label{performanceprecisiontradeoffs}
 The Part Load Ratio (PLR) for a single speed coil or a multiple speed coil at speed 1 is calculated using the equation below
 
 \begin{equation}
-{\left {PLR}\right} = \frac{SystemLoad-OutputOff}{OutputFull - OutputOff}
+PLR = \frac{SystemLoad-OutputOff}{OutputFull - OutputOff}
```

Comment:
The output of this change looks identical. Regardless, this is a good cleanup.

Reply:
Yeah, this is yet another instance where the LaTeX compiler issues a warning because the syntax is incorrect. If we really wanted to wrap PLR in "{}", we need to wrap it in `\left\{` and `\right\}`. However, I was assuming the wrapping of {} wasn't intentional. Cleaning up LaTeX issues where the compiler muddles through (so output doesn't change) but where the syntax as written is incorrect does really help with importing the LaTeX for the web-based documentation. Thanks!